Sources/TensorFlow/Bindings/TFTensorOperation.swift

```diff
@@ -12,6 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/// Opaque reference to a function that has been made callable by loading it
+/// into the runtime.
+public struct TensorFunctionPointer {
```

Comment:
Why does it need to be public?

Reply:
Alas, the protocol it is used in is public.

Reply:
I see. In that case, the type name should start with an underscore because it's an internal implementation detail.